### DISCUSSION TITLE
(Suggestion) "Minimap FPS Limiter" that can give hundreds of extra FPS for Xaero's minimap

### DISCUSSION THREAD

**Discussion by IdkWhatNameThingyToPut** [ACGaming/UniversalTweaks#586] [Tweak Suggestions]
Hello a "Minimap FPS Limiter" feature has already been done in a mod called XaeroPlus (https://www.curseforge.com/minecraft/mc-mods/xaeroplus) however that feature in the mod is only available for "1.19+",  and the developer of the mod XaeroPlus has said "Yeah I have no plans to backport it", they've also said "But if someone wishes to contribute a backport or reimplement it in their own mod both are fine with me.". This is all from a talk I had with them on an issue thiingy I made on their github which you can see here: https://github.com/rfresh2/XaeroPlus/issues/179

What I'm suggesting is for you to add your own type/implementation of a feature that does something like that from that mod because it seems really important to have if your using Xaero's minimap which is one of the best minimaps out there.

This is a video from the XaeroPlus mod's main page on CurseForge watch this to see how it gives heaps of additional FPS: https://www.youtube.com/watch?v=hIG-VyGQLao

In general it seems like a really good performance feature to have it looks like it gives hundreds of extra FPS. Really hoping you can add such a feature to your mod it'd be a good feature to have.

**Comment by ACGaming** [comment]
Guess you already got a solution via StellarCore.
